docs/installation/index.rst

```diff
@@ -30,9 +30,12 @@ For advanced installation and configuration, see the following topics.
 .. toctree::
     :maxdepth: 1
 
-    client-server
+    architecture
+    installing
     standalone
```

Comment:
`standalone` seems to be a relic from an older TOC.  It has the title "Using KSQL in Standalone Mode" (and talks about locked-down deployment mode).  This kind of info should nowadays be under "Configuring KSQL Server".

Reply:
I have removed this - applied to other PR: https://github.com/confluentinc/ksql/pull/974/commits/29ba8d94caa8f9ec22b75f380bca76bb6c48c39a